projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/checkout/delivery-address.ts

```diff
@@ -68,9 +62,8 @@ export function checkoutShippingAddressAccount(config: TabElement[]) {
     )}/**/addresses/delivery*`,
```

Comment:
Will this work now? 🤔
Since it's a wildcard that will probably catch both delivery address and delivery mode requests

Reply:
Right, my bad. I somehow missed the /addresses  🤦🏻‍♂️